shogun-data

```diff
@@ -0,0 +1 @@
+Subproject commit 9b12ffc6add78290c135025f913fab488657ff5d
```

Comment:
OK, I just merged #3219 which includes your hierarchical data.
This means you can remove the data version at all, and we can merge.

 However, make sure to understand this to have an easier life for the next cookbook ;)